src/raylib.h

```diff
@@ -306,6 +306,8 @@ typedef struct Camera3D {
     Vector3 up;             // Camera up vector (rotation over its axis)
     float fovy;             // Camera field-of-view apperture in Y (degrees) in perspective, used as near plane width in orthographic
     int projection;         // Camera projection: CAMERA_PERSPECTIVE or CAMERA_ORTHOGRAPHIC
+    int mode;               // Camera mode: CAMERA_FREE, CAMERA_FIRST_PERSON, CAMERA_THIRD_PERSON, CAMERA_ORBITAL or CUSTOM
```

Comment:
As commented, `mode` and `swingCounter` can be avoided from the struct.

Reply:
The `mode` will be removed and added as a parameter to `UpdateCamera(..)`.

I am not sure how to remove `swingCounter` though?
For the view bobbing we need to store some state, right? Are there other algorithms to achieve the same effect without state?